docs/source/containerization/kubernetes.mdx

```diff
@@ -1,246 +1,274 @@
+
+### Set up cluster
+
+Install the tools and provision the infrastructure for your Kubernetes cluster.
+
+> For example, see the [Setup from Apollo's Build a Supergraph tutorial](https://github.com/apollosolutions/build-a-supergraph/tree/main/01-setup#01---setup). It provides steps you can reference for gathering accounts and credentials for your cloud platform (GCP or AWS), provisioning resources, and deploying your subgraphs.
+
+(TODO: refine the steps between this #set-up-cluster and the next #set-up-self-hosted-router)
+
+### Set up self-hosted router
+
+Set up a self-hosted router by following the [first six steps of the self-hosted router quickstart](/graphos/quickstart/self-hosted), and stopping when you reach step [7. Deploy your router and connect clients](https://www.apollographql.com/docs/graphos/quickstart/self-hosted/#7-deploy-your-router-and-connect-clients).
+
+
+## Install router Helm chart
+
+You can install the router chart from either a local repository or the container registry.
+
+### Install chart from container registry
+
+* Login to the container registry by following the guide to [authenticating to the GitHub container registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-to-the-container-registry).
+
+* After logging in, verify your access to the registry by showing the latest router chart from `helm`:
+
+  ```bash
+  helm show values oci://ghcr.io/apollographql/helm-charts/router
+  ```
+
+* Install the OCI image from the GiHub container registry:
+
+  ```bash title="OCI registry"
+  helm install --set router.configuration.telemetry.metrics.prometheus.enabled=true --set managedFederation.apiKey="<graph-api-key>" --set managedFederation.graphRef="<graph-ref>" --create-namespace --namespace router-deploy router-test oci://ghcr.io/apollographql/helm-charts/router --version 1.0.0-rc.8 --values router/values.yaml
+  ```
+
+### Install Helm chart from local repository
+
+* Clone the [router repository](https://github.com/apollographql/router).
+
+* Change to the directory in your local repo with the router chart:
+
+  ```bash
+  cd <repo-directory>/helm/chart
+  ```
+
+* Install the local chart, `router`:
+
+  ```bash title="Local repository"
+  helm install --set router.configuration.telemetry.metrics.prometheus.enabled=true --set managedFederation.apiKey="<graph-api-key>" --set managedFederation.graphRef="<graph-ref>" --create-namespace --namespace router-deploy router-test router --values router/values.yaml
+  ```
+
+  (TODO: clean up and describe the used helm install flags)
+
+  ```
+  In both the following examples, we are using helm to install the router:
+  - into namespace "router-deploy" (create namespace if it doesn't exist)
+  - with helm install name "router-test"
+  - with support for prometheus enabled
+  ```
+
```

Comment:
I don't think we should document how to do this. It's better to use our OCI published chart.